src/Leaflet.Editable.js

```diff
@@ -189,8 +189,9 @@ L.Editable = L.Class.extend({
         return polygon;
     },
 
-    createMarker: function (latlng) {
-        var marker = new this.options.markerClass(latlng, {editOptions: {editTools: this}});
+    createMarker: function (latlng, markerOptions) {
+        var options = L.Util.extend({editOptions: {editTools: this}}, markerOptions);
```

Comment:
No need to create a new var here, let's use `options` as parameter and do

`options = L.Util.extend…`